woocommerce/woocommerce-android
# Add the shipping label creation info screen

This PR is an addition to the #3015, which brings the shipping label creation information screen (which can be reused for other use cases).

![image](https://user-images.githubusercontent.com/1522856/99676610-08ae9280-2a79-11eb-81bb-9b58ef109150.png)

**To test:**
1. Go to Orders
2. Tap on an order that allows shipping label creation
3. Tap on the "Learn more about creating shipping labels with your phone"
4. Notice an information screen appears
5. Tap on the "Learn more" link
6. Notice a website with the plugin information is opened in Chrome
